chia/full_node/mempool_check_conditions.py

```diff
@@ -132,40 +134,222 @@ def mempool_assert_my_amount(condition: ConditionWithArgs, unspent: CoinRecord)
     return None
 
 
-def get_name_puzzle_conditions(generator: BlockGenerator, max_cost: int, safe_mode: bool) -> NPCResult:
+def parse_aggsig(args: SExp) -> List[bytes]:
+    pubkey = args.first().atom
+    args = args.rest()
+    message = args.first().atom
+    if len(pubkey) != 48:
+        raise ValidationError(Err.INVALID_CONDITION)
+    if len(message) > 1024:
+        raise ValidationError(Err.INVALID_CONDITION)
+    return [pubkey, message]
+
+
+def parse_create_coin(args: SExp) -> List[bytes]:
+    puzzle_hash = args.first().atom
+    args = args.rest()
+    amount = args.first().atom
+    if len(puzzle_hash) != 32:
+        raise ValidationError(Err.INVALID_CONDITION)
+    amount_int = int_from_bytes(amount)
```

Comment:
You can just go `args.first().as_int()` here instead of importing `int_from_bytes`. Convenient, but no big deal.

Reply:
we use `int_from_bytes()` in a lot of places in this file, so I don't think I could get away with not importing that function, but I agree that it might be nicer to use that convenience function here.

Reply:
the original reason I separated pulling out the atom and the integer was because I intended to preserve the atom buffer when producing the return value. But then realized it's better to re-create it as a minimal representation